pom.xml

```diff
@@ -135,6 +140,21 @@
         <type>pom</type>
         <scope>import</scope>
       </dependency>
+      <dependency>
+        <groupId>org.jboss.narayana.jta</groupId>
+        <artifactId>narayana-jta</artifactId>
+        <version>5.6.3.Final-SNAPSHOT</version>
+      </dependency>
+      <dependency>
+        <groupId>org.jboss.narayana.jta</groupId>
+        <artifactId>tomcat-jta</artifactId>
+        <version>5.6.3.Final-SNAPSHOT</version>
+      </dependency>
```

Comment:
correct, I would just wait for the patch Karel (@sutaakar) was about to send for MariaDB. Once that is all in we should be good to release narayana and merge this.

@MarianMacik any other comments regarding tests on different containers/dbs or we are good there too?